### DISCUSSION TITLE
[BUG] Opening

### DISCUSSION THREAD

**Discussion by TTVNTYN** [bridge-core/bridge.#214] [Q&A] [answered]
**Summary**
Basically when i go to my downloads it says Bridge and i click on it and it wont ever open a application it just keeps me in files explorer and it confuses me as i cant open it properly How to fix this??
**To Reproduce**
1. Download Bridge
2. Open bridge from the file explorer
3. Observe as you cannot open application

**Observed behavior**
I downloaded and clicked on it but it only went to files explorer and did not open an app
**Expected behavior**
I expected to download bridge go to files extract it from the .zip file and it to be a .exe but no i cant open the app....

**Screenshots / File Attachments**
![When i open downloads](https://user-images.githubusercontent.com/66833541/84508306-feae8980-acb9-11ea-8eaa-c5b1664ec922.png)
![When i open bridge](https://user-images.githubusercontent.com/66833541/84508326-066e2e00-acba-11ea-9abb-1b57dcb17f0a.png)

**Platform (please complete the following information):**
 - OS: Windows 10
 - App Version: v1.7.0-pre6

**Additional context**
Please fix this problem soon as i really want to start coding

**Comment by solvedDev** [comment, marked as answer]
You downloaded the source code and not the latest release. Make sure to grab the .exe from here:
https://github.com/bridge-core/bridge./releases
